### PR TITLE
Add addPool to AMM creatorFacet and reinstate economy boot

### PR DESCRIPTION
**Pull request by michaelfig**
refs: #5375

## Description

- Put addPool on the AMM creatorFacet, primarily to un-tie knots in automated bootstrap; not intended for normal production use.
- Fix economy bootstrap, and add to CI.

### CODE REVIEW

File: packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js [hunk 1]
```diff
@@ -291,10 +291,38 @@ const start = async (zcf, privateArgs) => {
     }),
   );
 
-  /** @type {GovernedCreatorFacet<*>} */
-  const creatorFacet = makeGovernorFacet(
+  // TODO: pass this into makeAddPoolInvitation
```
Comment (turadg) [inline packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js]: I think this should be done before merge to DRY out the two addPools. They've already diverged in that [the other one updates metrics](https://github.com/Agoric/agoric-sdk/blob/0f4e0b53ffea8ea608f200752d8d342c0a2ce332/packages/run-protocol/src/vpool-xyk-amm/addPool.js#L97). cc @Chris-Hibbert

File: packages/run-protocol/src/vpool-xyk-amm/addPool.js [hunk 2]
```diff
@@ -41,8 +41,9 @@ export const makeAddIssuer = (zcf, isInSecondaries, brandToLiquidityMint) => {
         AssetKind.NAT,
         harden({ decimalPlaces: 6 }),
       ),
-      mint => {
-        zcf.saveIssuer(secondaryIssuer, keyword);
+      /** @param {ZCFMint} mint */
+      async mint => {
+        await zcf.saveIssuer(secondaryIssuer, keyword);
```
Comment (turadg) [inline packages/run-protocol/src/vpool-xyk-amm/addPool.js:46]: noting https://github.com/Agoric/agoric-sdk/issues/5407 to link this instance to that ticket

**Review by Chris-Hibbert — CHANGES_REQUESTED** [review]
This makes it possible to create a new empty pool, violating the requirement for a minimum funding. It also allows/requires the pool to be funded using `addLiquidity` rather than going through `addPoolAndLiquidityHandler`, which means that no liquidity will be extracted to the reserve.

**Comment by dckc** [comment]
@Chris-Hibbert , this no-fee `addPool` method is primarily intended to un-tie knots in automated bootstrap; it's not intended for normal production use. (I clarified the PR description to say as much.)

> This makes it possible to create a new empty pool, violating the requirement for a minimum funding.

The motivation for minimum funding was essentially spam prevention, wasn't it? This no-fee `addPool` method is on the creator facet, which is only available to the BLDer DAO, not to spammers. There's nothing inherently wrong with an empty AMM pool, is there?

> It also allows/requires the pool to be funded using `addLiquidity` rather than going through `addPoolAndLiquidityHandler`, which means that no liquidity will be extracted to the reserve.

Yes, this is a consideration that anyone with access to the AMM creator facet should be aware of. Is it a reason not to add the feature, though?

**Comment by dckc** [comment]
discussion with @Chris-Hibbert , @dtribble and co suggests:

 - try PSM?
 - try 0 minimum required liquidity?

**Comment by dckc** [comment]
obsolete in favor of #5441

**Pull request closed without merging**